web/src/portico/team.js

```diff
@@ -67,7 +67,9 @@ export default function render_tabs() {
     const template = _.template($("#contributors-template").html());
     const count_template = _.template($("#count-template").html());
     const total_count_template = _.template($("#total-count-template").html());
-    const contributors_list = page_params.contributors.filter((c) => exclude_bot_contributors(c));
+    const contributors_list = page_params.contributors
+        ? []
+        : page_params.contributors.filter((c) => exclude_bot_contributors(c));
```

Comment:
Hmm, is this check inverted though?

Is the list empty, or are we failing to pass it in development? Might be the latter is the issue.

Reply:
Yeah, my bad. This is just not working in development.